update location of docker image

andrewhsu opened the pull request:
Change from: `vectorized` to `redpandadata`

Related issue: https://github.com/redpanda-data/devprod/issues/378

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/redpanda-data/documentation?pullRequest=1103) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/redpanda-data/documentation?pullRequest=1103) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/redpanda-data/documentation?pullRequest=1103) it.</sub>

Deflaimun commented:
Related PR #1069 Issue #644

shorgi commented:
@andrewhsu - within the last two weeks we added `versioned_docs/version-22.3`, can you please make your edits there as well? Your changes look good otherwise.

andrewhsu commented:
> @andrewhsu - within the last two weeks we added `versioned_docs/version-22.3`, can you please make your edits there as well? Your changes look good otherwise.

done. i added 3 commits.

kbatuigas commented on modules/ROOT/attachments/cloud-dataplane-api.yaml:
```diff
@@ -1227,7 +1227,7 @@ components:
           additionalProperties:
             type: string
-          description: Optional list of tags to attach to a pipeline.
+          description: Optional lList of tags to attach to a pipeline.
```
```suggestion
          description: Optional list of tags to attach to a pipeline.
```

kbatuigas commented on modules/ROOT/attachments/cloud-dataplane-api.yaml:
```diff
@@ -1178,7 +1178,7 @@ components:
         url:
           description: |-
-            URL to connect to the pipeline, for example, using http_server.
+            URL to connect to the pipeline, e.g. via http_server.
```
```suggestion
            URL to connect to the pipeline, for example, using http_server.
```
